testing/src/main/java/io/stargate/it/grpc/GrpcAuthenticationTest.java

```diff
@@ -15,13 +15,15 @@
  */
 package io.stargate.it.grpc;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 import io.grpc.StatusRuntimeException;
+import io.stargate.proto.QueryOuterClass;
 import io.stargate.proto.QueryOuterClass.Query;
 import org.junit.jupiter.api.Test;
 
-public class AuthenticationTest extends GrpcIntegrationTest {
+public class GrpcAuthenticationTest extends GrpcIntegrationTest {
```

Comment:
Just to make it easier to invoke (unique name) and more easily recognize for cases where simple (instead of FQN) name used.